apps/web/components/forgot-password.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from "react";
+import Link from "next/link";
+import { Button } from "@documenso/ui";
+import Logo from "./logo";
+import { ArrowLeftIcon } from "@heroicons/react/24/outline";
+import { FormProvider, useForm } from "react-hook-form";
+import { toast } from "react-hot-toast";
+
+interface ForgotPasswordForm {
+  email: string;
+}
+
+export default function ForgotPassword() {
+  const methods = useForm<ForgotPasswordForm>();
```

Comment:
Typically I prefer destructuring here, but that's personal preference and I won't hold it against you if you don't.

Reply:
Destructuring means `methods` wont be available for FormProvider so I'm removing the FormProvider as well.